src-java/connecteddevices-topology/connecteddevices-storm-topology/src/test/java/org/openkilda/wfm/topology/connecteddevices/service/PacketServiceTest.java

```diff
@@ -130,6 +135,20 @@ public void testHandleLldpDataDifferentTimeOnUpdate() throws InterruptedExceptio
         assertNotEquals(devices.iterator().next().getTimeFirstSeen(), devices.iterator().next().getTimeLastSeen());
     }
 
+    @Test
+    public void testHandleArpDataDifferentTimeOnUpdate() throws InterruptedException {
+        // create
+        packetService.handleArpData(createArpInfoData());
+
+        Thread.sleep(10);
```

Comment:
Why do you need this sleep?

Reply:
this test checks that create and update operations has different timestams. MessageData class (parent of ArpInfoData) sets timestamp in constructor by `System.currentTImeMills()`. Some time this test is too fast and timestamp in create and update requests are equal. to guarantee different timestamp I added `sleep(10)` here